freeipa/src/main/resources/schema/app/20190516151400_add_tenant_to_stack.sql

```diff
@@ -12,21 +12,16 @@
 ALTER TABLE stack
     ADD tenant VARCHAR(255);
 
-CREATE UNIQUE INDEX tenant_uindex
+CREATE INDEX tenant_uindex
```

Comment:
naming convention looks to be:
xxxx_idx for normal index
xxxx_uindex for unique index

please adjust these accordingly